misc/org.projecthamster.hamster-gtk.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=hamster-gtk
```

Comment:
IMHO, the user should not care about the frontend library, their software repository should offer them the most fitting variant. The details might be listed in the About dialogue.